admin-app/webpack.config.js

```diff
@@ -78,7 +78,8 @@ module.exports = {
   devServer: {
     historyApiFallback: true,
     noInfo: true,
-    overlay: true
+    overlay: true,
+    disableHostCheck: true
```

Comment:
This allows "hot" reloading on the WP admin, when a certain version of Webpack throws a Wrong Origin error.